ai2thor/server.py

```diff
@@ -295,6 +295,7 @@ def __init__(self, width, height, depth_format=DepthFormat.Meters, add_depth_noi
         self.sequence_id = 0
         self.started = False
         self.client_token = None
+        self.unity_proc = None
```

Comment:
This was added to allow one to debug with the Unity editor open from Python. That is,
```python
from ai2thor.controller import Controller
from ai2thor.wsgi_server import WsgiServer
Controller(start_unity=False, port=8200, server_class=WsgiServer)
```